Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -2258,6 +2269,18 @@ confirmingLottieView push state =
           ]
     ]
 
-genderBanner :: forall w . (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM (Effect Unit) w
-genderBanner push state =
-  Banner.view (push <<< GenderBannerModal) (genderBannerConfig state)
\ No newline at end of file
+checkAppVersion :: forall action. (action -> Effect Unit) -> action -> HomeScreenState -> Flow GlobalState Unit
+checkAppVersion push action state = do
+  if os == "IOS" then do 
+      apires <- getAppStoreVersion (getValueFromConfig "appId")
+      versionName <- pure $ unsafePerformEffect getVersionName
+      case apires of
+            Right (AppStoreRes response) -> do
+              result <- pure $ head response.results
+              (AppDetails app) <- pure $ fromMaybe dummyAppDetail result
+              let showBanner = startUpdate "CHECK_UPDATE_FUNCTION_IN_IOS"
+              if(versionName < app.version && showBanner) then doAff do liftEffect $ push $ action else pure unit
```

Comment:
replace this `doAff do liftEffect` with `liftFlow` in all place

Reply:
Done